core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2339,6 +2339,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_METASTORE_DIR_INODE =
+      stringBuilder(Name.MASTER_METASTORE_DIR_INODE)
+          .setDefaultValue(String.format("${%s}", Name.MASTER_METASTORE_DIR))
+          .setDescription("The inode metastore work directory. "
+              + "Only some metastores need disk.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_METASTORE_DIR_BLOCK =
+      stringBuilder(Name.MASTER_METASTORE_DIR_BLOCK)
+          .setDefaultValue(String.format("${%s}", Name.MASTER_METASTORE_DIR))
+          .setDescription("The block metastore work directory. "
+              + "Only some metastores need disk.")
```

Comment:
```suggestion
          .setDescription("If the metastore is ROCKS, this property controls where the RocksDB stores block metadata. This property defaults to " + Name.MASTER_METASTORE_DIR + ". And it can be used to change block metadata storage path to a different disk to improve RocksDB performance.")
```